http-core/src/main/scala/org/apache/pekko/http/impl/model/parser/UriParser.scala

```diff
@@ -365,7 +365,7 @@ private[http] final class UriParser(
 
   ///////////// helpers /////////////
 
-  private def appendLowered(): Rule0 = rule { run(sb.append(CharUtils.toLowerCase(lastChar))) }
+  private def appendLowered(): Rule0 = rule { run(sb.append(CU.toLowerCase(lastChar))) }
```

Comment:
can we assume `lastChar` is 7-bit ascii here? or are we OK with letting other characters through?